sockeye/train.py

```diff
@@ -219,10 +219,16 @@ def create_data_iters_and_vocabs(args: argparse.Namespace,
                                  max_seq_len_target: int,
                                  shared_vocab: bool,
                                  resume_training: bool,
-                                 output_folder: str) -> Tuple['data_io.BaseParallelSampleIter',
-                                                              'data_io.BaseParallelSampleIter',
-                                                              'data_io.DataConfig',
-                                                              List[vocab.Vocab], vocab.Vocab]:
+                                 validation_sources: Optional[List[str]] = None,
```

Comment:
I probably need to look better into this, but why is this suddenly getting all these new arguments?

Reply:
I had to pull out arguments from `args` that are used in training and not in scoring.